### PR TITLE
Improve fixture generation

### PR DESCRIPTION
**Pull request by javiereguiluz**
* Add at least 1 tag to every blog post, to make the design more consistent
* Don't cut out the phrases of the post summary.

### CODE REVIEW

File: src/AppBundle/DataFixtures/ORM/PostFixtures.php [hunk 1]
```diff
@@ -91,11 +91,7 @@ public function getDependencies()
 
     private function addRandomTags(Post $post)
     {
-        if (0 === $count = mt_rand(0, 3)) {
```
Comment (stof) [inline src/AppBundle/DataFixtures/ORM/PostFixtures.php]: this means we don't have any post without tags in the fixtures. Is it expected ? Tagging posts is optional

**Comment by stof** [comment]
> Add at least 1 tag to every blog post, to make the design more consistent

The validation of posts does not require having at least 1 tag. So if the design does not support such case, it means that there is a bug in the code, not that fixtures should be changed

**Comment by javiereguiluz** [comment]
I've changed the fixtures again: posts can have 0 tags ... but the first post now always has 2 tags, to make the application look better.

**Comment by stof** [comment]
👍

**Pull request closed without merging**